.travis.yml

```diff
@@ -8,10 +8,22 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
+  - go get -u github.com/HewlettPackard/gas
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
+  # GAS command for SDL scanning
+  - (cd arm && gas -fmt=json -out=results.json ./... > /dev/null)
+  - cat arm/results.json
+  - test $(grep -c "\"severity\":\s*\"HIGH\"" arm/results.json) -lt 8
+  - (cd management && gas -fmt=json -out=results.json ./... > /dev/null)
+  - cat management/results.json
+  - test $(grep -c "\"severity\":\s*\"HIGH\"" management/results.json) -lt 8
```

Comment:
@mbearup I have updated the thresholds. ```Test123``` in finding is dummy password so we can ignore that.
@ahmetalpbalkan take a look.